jts-transaction-service/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM fedora:22
+MAINTAINER Gytis Trikleris <gytis@redhat.com>
+WORKDIR /home
+
+ENV NARAYANA_PORT 4710
+ENV NARAYANA_VERSION 5.2.1.Final
+ENV NARAYANA_HOME /home/narayana
+ENV JACORB_LOG_LEVEL 3
+
+RUN dnf install -y java wget unzip \
+    && wget http://www.jboss.org/jbosstm/downloads/$NARAYANA_VERSION/binary/narayana-full-$NARAYANA_VERSION-bin.zip \
+    && unzip narayana-full-$NARAYANA_VERSION-bin.zip \
+    && mv /home/narayana-full-$NARAYANA_VERSION $NARAYANA_HOME \
+    && rm narayana-full-$NARAYANA_VERSION-bin.zip
+
+ENV JAVA_HOME /etc/alternatives/jre
+
+COPY docker-entrypoint.sh /home/docker-entrypoint.sh
+
+ENTRYPOINT ["/home/docker-entrypoint.sh"]
+CMD source $NARAYANA_HOME/jts-jacorb-setup-env.sh; \
+    eval java -Dcom.arjuna.orbportability.common.OrbPortabilityEnvironmentBean.bindMechanism=NAME_SERVICE \
+        -DObjectStoreEnvironmentBean.objectStoreDir=/home/tx-object-store \
+        $NARAYANA_OPTS \
+        -DOAPort=$NARAYANA_PORT \
+        -Djacorb.log.default.verbosity=$JACORB_LOG_LEVEL \
+        com.arjuna.ats.jts.TransactionServer -recovery -ORBInitRef.NameService=$NAME_SERVER_URL
+EXPOSE 4710
```

Comment:
It's a good idea to move stuff that will not change (`EXPOSE` instruction in this case) to the top of the `Dockerfile`.